e2e/credentials-server.e2e.spec.ts

```diff
@@ -255,4 +276,37 @@ describeWithCreds('Credentials server', () => {
     // Don't bother trying to test erasing a non-existent credential, because the
     // behavior is all over the place. Fails with osxkeychain, succeeds with wincred.
   });
+
+  test.describe('should be able to detect errors', () => {
+    const bobsURL = 'https://bobs.fish/bait';
+
+    test('it should complain when no ServerURL is given', async() => {
+      const body: Record<string, string> = {};
+      const { stdout, stderr } = await rdctlCredWithStdin('store', JSON.stringify(body));
+
+      expect(stdout).toContain('no credentials server URL');
+      expect(stderr).toContain('Error: exit status 22');
```

Comment:
(I'm happy with it as-is, just comment for next PR:)

```suggestion
      await expect(rdctlCredWithStdin('store', JSON.stringify(body))).resolves.toMatchObject({
        stdout: expect.stringContaining('no credentials server URL'),
        stderr: expect.stringContaining('Error: exit status 22'),
      });
```